charts/kube-state-metrics/values.yaml

```diff
@@ -181,6 +181,17 @@ podSecurityPolicy:
 
   additionalVolumes: []
 
+## Configure network policy for kube-state-metrics
+networkPolicy:
+  enabled: false
+  # egress:
+  # - {}
+  # ingress:
+  # - {}
+  # podSelector:
+  #   matchLabels:
+  #     app: kube-state-metrics
```

Comment:
This should likely use the standardized app. kubernetes.io/name label

Reply:
Sure, I changed that :)